tests/integration/table_details_test.go

```diff
@@ -0,0 +1,134 @@
+package integration
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"testing"
+
+	"github.com/jackc/pgx"
+	"github.com/perangel/warp-pipe/db"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGenerateTablesList(t *testing.T) {
+	// Re-used test case elements
+	defaultTableCreateQueries := []string{
+		`CREATE TABLE "test1" (id1 INT PRIMARY KEY, content TEXT);`,
+		`CREATE TABLE "test2" (id2 UUID PRIMARY KEY, content BYTEA);`,
+		`CREATE TABLE "test3" (id3 BIGINT PRIMARY KEY, content TIMESTAMP WITH TIME ZONE);`,
+	}
+	crdbDefaultExpected := []db.Table{
+		{Name: "test1", Schema: "public", PKeyName: "primary", PKeyFields: map[int]string{1: "id1"}},
+		{Name: "test2", Schema: "public", PKeyName: "primary", PKeyFields: map[int]string{1: "id2"}},
+		{Name: "test3", Schema: "public", PKeyName: "primary", PKeyFields: map[int]string{1: "id3"}},
+	}
+	psqlDefaultExpectedTables := []db.Table{
+		{Name: "test1", Schema: "public", PKeyName: "test1_pkey", PKeyFields: map[int]string{1: "id1"}},
+		{Name: "test2", Schema: "public", PKeyName: "test2_pkey", PKeyFields: map[int]string{1: "id2"}},
+		{Name: "test3", Schema: "public", PKeyName: "test3_pkey", PKeyFields: map[int]string{1: "id3"}},
+	}
```

Comment:
The only difference between CRDB and PSQL here is that the name of the primary key constraint is different; PSQL uses `tablename_pkey`, while CRDB just uses `primary`

Reply:
I couldn't find anywhere in the codebase where we even reference `PKeyName` (might be able to deprecate the field) so I think it's a safe drift.